app/views/admin/_header.html.erb

```diff
@@ -54,6 +54,15 @@
       <li><%= link_to t(".nav.banners"), admin_banners_path %></li>
     <% end %>
 
+    <% if policy(SupportNotice).index? %>
+      <li class="dropdown">
+        <%= link_to t(".nav.notices.notices"), admin_support_notices_path %>
+        <ul class="menu">
+          <li><%= link_to t(".nav.notices.support"), admin_support_notices_path %></li>
+        </ul>
```

Comment:
Since both of these links point to `admin_support_notices_path` it's okay to skip the `ul` menu and the `dropdown` class. (Settings and Banners on the lines above are also just a top-level items.)

Reply:
:+1: I've updated the issue to match.